gold-plugins/two-step-auth.js

```diff
@@ -111,41 +131,90 @@ exports.commands = {
 		setup: function (target, room, user) {
 			if (!user.named) return this.errorReply("You must be logged in to use this command.");
 			if (!user.registered) return this.errorReply("You cannot setup two-step authentication on an account that isn't registered.");
+			if (Gold.userData[user.userid] && (Gold.userData[user.userid].email || Gold.userData[user.userid].twostepauth)) return this.errorReply("This account already has two-step authentication enabled.");
```

Comment:
I can't think of how ``Gold.userData[user.userid]`` here could be undefined...

There are checks above to make sure the user is logged in and not on a guest account, so I don't think that part of the if statement is necessary... Correct me if I'm wrong?